plugins/module_utils/entrust_cagw/api.py

```diff
@@ -0,0 +1,347 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is licensed under the
+# Modified BSD License. Modules you write using this snippet, which is embedded
+# dynamically by Ansible, still belong to the author of the module, and may assign
+# their own license to the complete work.
+#
+# Copyright (c), Entrust Datacard Corporation, 2020
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
```

Comment:
You also need to add
```suggestion
# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
# SPDX-License-Identifier: BSD-2-Clause
```
(See the failing extra sanity tests, and the failing reuse test.)

Reply:
I changed the license to GPL-3.0 and added below too
 # SPDX-License-Identifier: GPL-3.0-or-later